app/src/main/java/com/example/drawingactivity/CustomColorViewModel.kt

```diff
@@ -36,11 +37,31 @@ class CustomColorViewModel : ViewModel() {
         currentBitmap = b
     }
 
+    var isDialogOpen = false
+    var layerName: String? = null
+    private var layerNumber: Int = 1
+
+    fun makeLayerNameHint(s: String): String {
+        return "$s $layerNumber"
```

Comment:
Resource

Reply:
To clarify, the reason this should be a resource also is that your order won't be the same in every language

Reply:
Deleted this function as I am now using a formatted string xml. There is a lot of consideration that has to go into localization for strings, is what I learned.